tests/test_shadow_fork.py

```
@@ -76,25 +76,32 @@ def test_create_fails_if_base_branch_missing(self, tmp_path):
         """Test that create fails if base branch doesn't exist."""
         fork = ShadowFork(repo_root=tmp_path)
 
-        with patch.object(fork, "_run_git") as mock_git:
+        # Mock _resolve_base_ref to return a reference that causes checkout to fail
+        # This isolates us from the internal logic of _resolve_base_ref
+        with patch.object(fork, "_run_git") as mock_git, \
+             patch.object(fork, "_resolve_base_ref", return_value="nonexistent") as mock_resolve:
```

Ruff (0.14.14)

[error] Local variable `mock_resolve` is assigned to but never used

Remove assignment to unused variable `mock_resolve`

(F841)